src/admin-views/list.php

```diff
@@ -40,7 +40,7 @@
 		 * @param string $ticket_provider
 		 */
 		if ( apply_filters( 'tribe_tickets_current_user_can_delete_ticket', true, $ticket->ID, $ticket->provider_class ) ) {
-			$controls[] = sprintf( '<span><a href="#" attr-provider="%1$s" attr-ticket-id="%2$s" id="ticket_delete_%2$s" class="ticket_delete">' . esc_html__( 'Delete', 'event-tickets' ) . '</a></span>', $ticket->provider_class, $ticket->ID );
+			$controls[] = sprintf( '<span><a href="#" attr-provider="%1$s" attr-ticket-id="%2$s" onClick="return confirm(\'Are you sure you want to delete This ticket?\')" id="ticket_delete_%2$s" class="ticket_delete">' . esc_html__( 'Delete', 'event-tickets' ) . '</a></span>', $ticket->provider_class, $ticket->ID );
```

Comment:
A good rule of thumb is to avoid the use of event attributes in HTML. Instead, we prefer to keep stuff like this in JS files because it allows for some amount of organization, minification, and customization. This click event should probably live in `src/resources/js/tickets.js`.